### PR TITLE
Chore: integrate zod typecasting pt. 2

### PR DESCRIPTION
**Pull request by ghost**
Updated TSConfig to use env.mjs
Updated env variables in 4 files

### CODE REVIEW

File: components/admin-dashboard/whitelisted-addresses.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-export const whitelistedAddresses = process.env
-  .NEXT_PUBLIC_WHITELISTED_ADDRESSES
-  ? process.env.NEXT_PUBLIC_WHITELISTED_ADDRESSES.split(",")
-  : [];
+import { env } from "../../config/env.mjs";
```
Comment (gaelicgod) [inline components/admin-dashboard/whitelisted-addresses.ts:1]: use the consistent import, the `@/config/env.mjs`

File: components/admin-dashboard/whitelisted-addresses.ts [hunk 2]
```diff
@@ -1,4 +1,3 @@
...
+
+export const whitelistedAddresses = env.NEXT_PUBLIC_WHITELISTED_ADDRESSES ?? [];
```
Comment (gaelicgod) [inline components/admin-dashboard/whitelisted-addresses.ts:3]: why do you need a default value check here?

File: components/admin-dashboard/whitelisted-addresses.ts (hunk 1 above)
Comment (EsterB12) [reply components/admin-dashboard/whitelisted-addresses.ts]: I have updated this!

File: components/admin-dashboard/whitelisted-addresses.ts (hunk 2 above)
Comment (EsterB12) [reply components/admin-dashboard/whitelisted-addresses.ts]: I probably don't ':D

File: components/admin-dashboard/whitelisted-addresses.ts [hunk 3]
```diff
@@ -1,4 +1,3 @@
-export const whitelistedAddresses = process.env
-  .NEXT_PUBLIC_WHITELISTED_ADDRESSES
-  ? process.env.NEXT_PUBLIC_WHITELISTED_ADDRESSES.split(",")
-  : [];
+import { env } from "@/config/env.mjs";
+
+export const whitelistedAddresses = env.NEXT_PUBLIC_WHITELISTED_ADDRESSES;
```
Comment (gaelicgod) [inline components/admin-dashboard/whitelisted-addresses.ts:3]: @EsterB12 do you not need the .split() anymore? is this in the env.mjs file?

Comment (EsterB12) [reply components/admin-dashboard/whitelisted-addresses.ts:3]: yes, its in the env.mjs!

**Review by gaelicgod — APPROVED** [review]

**Pull request merged**